index.ts

```diff
@@ -17,7 +17,9 @@ export default class PopoverGallery extends HTMLElement {
                 <img src="${linkEl.getAttribute("href")}" alt="${altText}">
                 <figcaption>${caption}</figcaption>
               </figure>`
-          : `<img src="${linkEl.getAttribute("href")}" alt="${altText}">`;
+          : `<figure>
+              <img src="${linkEl.getAttribute("href")}" alt="${altText}">
+            </figure>`;
```

Comment:
Since the `<figure>` is present in both cases, it (and `<img>`), they can be hard-coded into line 31.

This ternary expression should only be used to wrap `caption` in `<figcaption>`.